src/app/components/home/home.component.ts

```diff
@@ -14,47 +21,66 @@ export class HomeComponent implements OnInit {
   panelOpenState = false;
   realm: Realm;
   realms: Realm[];
+  users: User[];
+  admin: User;
+  private subscription: Subscription;
 
-  constructor(private router: Router, private realmService: RealmService, private adminAuthService: AdminAuthService) {
+  constructor(private router: Router,
+              private realmService: RealmService,
+              private adminAuthService: AdminAuthService,
+              private roleService: RoleService,
+              private clientService: ClientService,
+              private snackBar: SnackBarService,
+              private userService: UserService,
+              private groupsService: GroupService) {
   }
 
   ngOnInit(): void {
+    this.admin = JSON.parse(localStorage.getItem('token')).username;
```

Comment:
ar trebui sa exista un BehaviorSubject in token service de unde poti lua token-ul